src/ipc-handlers.ts

```diff
@@ -639,8 +640,8 @@ export function logRendererMessage(
 	writeLogToFile( level, processId, ...args );
 }
 
-export async function authenticate( _event: IpcMainInvokeEvent ): Promise< void > {
-	return oauthClient.authenticate();
+export function authenticate( _event: IpcMainInvokeEvent ): void {
```

Comment:
```suggestion
export function authenticate( _event: IpcMainInvokeEvent ) {
```
Maybe someone disagrees with me, but I don't think it's necessary to explicitly add a `void` return type here.